test/config.json

```diff
@@ -16,19 +16,21 @@
       "browsers": [
         {
           "browserName": "Firefox",
-          "browser_version": "83",
+          "browser_version": "latest",
           "os": "Windows",
           "os_version": "10",
           "resolution": "1920x1080",
-          "remote": true
+          "remote": true,
+          "name": "multipleBrowsersDocumentUpload"
         },
         {
           "browserName": "Edge",
-          "browser_version": "87",
+          "browser_version": "89",
```

Comment:
Maybe I remembered it wrongly, but we can't set `"browser_version"` to `"latest"` for Edge also like for Firefox?

Reply:
We can but I noticed that setting it to latest it was throwing some exceptions, basically it looks like there is an issue on BrowserStack side where they may be using an incorrect version of the MSEdgeDriver on their side, so I didnt want us to unnecessarily see exceptions on our side.

Reply:
Ah, ok. Makes sense then to not use that setting.